src/sentry/constants.py

```diff
@@ -168,7 +168,6 @@ def get_all_languages():
     'password',
     'secret',
     'passwd',
-    'authorization',
```

Comment:
This wasn't doing anything since these values are used in a containment test and there's `auth` below